lib/spack/docs/getting_started.rst

```diff
@@ -16,7 +16,7 @@ Prerequisites
 Spack has the following minimum requirements, which must be installed
 before Spack is run:
 
-#. Python 2 (2.6 or 2.7) or 3 (3.4 - 3.7) to run Spack
+#. Python 2 (2.6 or 2.7) or 3 (3.5 - 3.8) to run Spack
```

Comment:
Do we want to support Python 3.4? I don't think we can claim that we support it unless we test it.

Reply:
No, we removed Python 3.4 a while ago - I think this is just a leftover of what was there before.